src/System.Collections.Specialized/src/System/Collections/Specialized/ListDictionary.cs

```diff
@@ -483,9 +483,9 @@ public void Reset()
         [System.Runtime.CompilerServices.TypeForwardedFrom("System, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089")]
         public class DictionaryNode
         {
-            public object key; // Do not rename (binary serialization)
-            public object value; // Do not rename (binary serialization)
-            public DictionaryNode next; // Do not rename (binary serialization)
+            public object key = null!; // Do not rename (binary serialization)
```

Comment:
This is a bit odd, but if DictionaryNode isn't in the ref / actually exposed publicly, it's fine.

Reply:
yes its not exposed in the ref